slyd.conf

```diff
@@ -3,5 +3,7 @@ start on vagrant-mounted
 stop on runlevel [!2345]
 
 chdir /vagrant/slyd
-exec bin/slyd -p 8000 --root ./dist
+script
+    PYTHONPATH=/vagrant/slyd:/vagrant/slybot bin/slyd -p 9002 --root ./dist
```

Comment:
I needed to change this here to 9001, otherwise the service wouldn't show up on localhost:9001